src/main/java/com/fontys/api/service/TournamentService.java

```diff
@@ -37,15 +43,15 @@ public Tournament createTournament(String name, String description, Integer owne
     @Transactional
     public Tournament updateTournament(Integer id, String name, String description, Integer ownerId, Integer numberOfTeams) throws InvalidAttributeValueException {
 
-        validateTournament(id);
+        Tournament oldTournament = validateTournament(id);
```

Comment:
There also is no reason to use oldTournament instead of tournament

Reply:
The code has been changed according to your feedback.

Reply:
Resolving this becuase there in a newer version